README.md

```diff
@@ -10,42 +10,22 @@ We are going to set up our ParallelCluster infrastructure as below:
-
-
-### NAT gateway
-A Network Address Translation (NAT) gateway is required for compute nodes in the private subnet to connect to service outside (i.e., web access for essential software updates or packages) the VPC. This is a one-way connection such that outside services cannot connect to the nodes inside the private subnet. For convenience, choose NAT gateway option during the time you create the VPC, so that a NAT gateway is created automatically. In this example, it is created while we set up VPC as shown in [the VPC setup instructions](./examples/general/network/vpc-setup.md).
+A ParallelCluster requires a VPC. First, you must have these component created and configured before creating the cluster. [Here](./examples/general/network/vpc-setup.m) is the instruction to create a VPC. This VPC has two subnets as described in [this diagram](https://docs.aws.amazon.com/parallelcluster/latest/ug/network-configuration-v3.html#network-configuration-v3-two-subnets "Network configuration"). Second, after VPC is created, follow [this instruction](./examples/general/network/subnet-setup.md) to configure the public subnet. 
 
 ### Key pair
-You also need a key pair. You may use an existing one. But if you wish to create a new key pair, [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/create-key-pairs.html#having-ec2-create-your-key-pair "Create key pair") is the instruction.
-
-### AMI
-You also need a ParallelCluster AMI to run on your cluster. Follow the steps below:
-
-1. Find a list of available ParallelCluster AMI in [here](https://github.com/aws/aws-parallelcluster/blob/v2.11.7/amis.txt). For the example here, we will use Amazon Linux 2 AMI. You may find the AMI ID based on the region accessible by your account. 
-
-2. Once the AMI is chosen, make a note of the AMI ID; you will need this AMI to create a ParallelCluster. 
-
-3. After the cluster is created, you will need to run a post-installation script.
+You also need a key pair. You may use an existing one. But if you wish to create a new key pair, [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/create-key-pairs.html#having-ec2-create-your-key-pair "Create key pair") is the instruction. A key is pair is needed so you may SSH into the head node of the cluster.
 
 Details and steps are provided in [here](./examples/general/ami/ami_setup.md)
 
-
 ### AWS Parallel Cluster Python package
 
 This is needed in a local environment (i.e., your Mac/PC desktop with a CLI terminal or an AWS Cloud9) where you issue the command to launch the creation process for your HPC environment in AWS. See [this](https://docs.aws.amazon.com/parallelcluster/latest/ug/install-v3-virtual-environment.html) for details on how to install this package. After the installation. A Python virtual environment will be created. Activate this particular virtual environment. 
```

Comment:
Replace "This" with "AWS Parallel Cluster Python package"

Reply:
"After the installation, a Python virtual environment will be created. Activate this virtual environment before running any pcluster command."

Reply:
Add in new commit